OmsAgent/omsagent.py

```diff
@@ -1059,6 +1059,8 @@ def detect_multiple_connections(workspace_id):
         # default encoding in python is ascii in python < 3
         if sys.version_info < (3,):
             output = utfoutput.decode('utf8').encode('utf8')
+        else:
+            output = utfoutput
```

Comment:
not related rt?

Reply:
yes, but is a real issue surfaced by Pylance; if you prefer a separate PR I can remove this change